profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.strongarm.inc

```diff
@@ -0,0 +1,295 @@
+  $strongarm->name = 'additional_settings__active_tab_newsroom_item';
+  $strongarm->value = 'edit-menu';
+  $export['additional_settings__active_tab_newsroom_item'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_anonymous_newsroom_selection';
+  $strongarm->value = '0';
+  $export['comment_anonymous_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_default_mode_newsroom_selection';
+  $strongarm->value = 1;
+  $export['comment_default_mode_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_default_per_page_newsroom_selection';
+  $strongarm->value = '50';
+  $export['comment_default_per_page_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_form_location_newsroom_selection';
+  $strongarm->value = 1;
+  $export['comment_form_location_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_newsroom_selection';
+  $strongarm->value = '1';
+  $export['comment_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_preview_newsroom_selection';
+  $strongarm->value = '1';
+  $export['comment_preview_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'comment_subject_field_newsroom_selection';
+  $strongarm->value = 1;
+  $export['comment_subject_field_newsroom_selection'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'date_format_date_only';
+  $strongarm->value = 'd/m/Y';
+  $export['date_format_date_only'] = $strongarm;
+
+  $strongarm = new stdClass();
+  $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
+  $strongarm->api_version = 1;
+  $strongarm->name = 'date_format_minimal';
```

Comment:
actually it's the same format than 'date_format_date_only' - could you merge them into one?